.github/workflows/code-quality.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       - name: Check Pull Request Size
         run: |
           git fetch origin ${{ github.event.pull_request.base.ref }} --quiet # Need to manually fetch base branch in CI
-          python ./.github/scripts/commit-filesize-diff-summary.py --limit 2MB origin/${{ github.event.pull_request.base.ref }}..HEAD
+          python ./.github/scripts/commit-filesize-diff-summary.py --limit 4MB origin/${{ github.event.pull_request.base.ref }}..HEAD
```

Comment:
While this will unblock this PR, let us revert it once this PR gets merged.

Reply:
Will raise the PR right after.